lib/run-compilation.js

```diff
@@ -41,12 +40,12 @@ module.exports = function runCompilation (options, compiler, done) {
 
   compiler.plugin('after-emit', function afterEmit (compilation, callback) {
     if (warnings.length) {
-      compilation.warnings.push(chalk.yellow(options.formatter(warnings)));
+      compilation.warnings.push(new Error(options.formatter(warnings)));
```

Comment:
Should warnings really be wrapped in `Error` objects too? Does this break builds if you only have warnings, but not errors?